src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -4,6 +4,14 @@
 import sys
 sys.path.append(os.getenv('HELIX_WORK_ROOT'))
 import copy
+"""
+ This is a temporary solution to gather h/w info from machine
+ the package is installed on end machines running perf
+ However this runs on a limited number of platforms
+ (but supports all the flavors we currently use for testing)
+ moving forward we will need a truly cross-plat solution here
+"""
+from cpuinfo import cpuinfo
```

Comment:
All the other lines are just "import". Any particular reason this line is "from cpuinfo import cpuinfo"?